test/test-expression-builder.rb

```diff
@@ -303,6 +303,14 @@ def test_match
                      result.collect {|record| record.key.key}.sort)
       end
 
+      def test_match_uppercase
+        result = @users.select do |record|
+          record["name"] =~ /\ASu/i
+        end
```

Comment:
It seems that `/\ASu/i` is not expected, `/Su/i` is intended test case because "Shiro SUZUKI" is used.

Reply:
We should put this after `test_end_of_text` and use name `test_option`.